appscale/tools/appscale_tools.py

```diff
@@ -51,6 +57,12 @@ def async_layout_upgrade(ip, keyname, script, error_bucket, verbose=False):
     error_bucket.put(ssh_error)
 
 
+MIN_FREE_DISK_DB = 40.0
```

Comment:
I agree with @cdonati: the server side should returns those warning. I would be even more generic: the server site should return warnings (as in error codes/strings) not even the parameters to check for errors. The tools should just report whatever warning it reads. This would get a long way once we get hermes fully implemented :)